package.json

```diff
@@ -10,7 +10,7 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "tape ./tests/*.test.js | tap-spec",
+    "test": "tape ./tests/*/*/*.test.js | tap-spec && tape ./tests/*/*.test.js | tap-spec",
```

Comment:
Rather than having to run our tests separately we could use a test runner file which requires in each of our tests - we could do this hierarchically like in this project: https://github.com/cul-2016/quiz/blob/staging/test/back/db/runner.js

Reply:
Done :)